src/netbase.cpp

```diff
@@ -40,7 +40,7 @@ static proxyType proxyInfo[NET_MAX];
 static proxyType nameProxy;
 static CCriticalSection cs_proxyInfos;
 int nConnectTimeout = DEFAULT_CONNECT_TIMEOUT;
-bool fNameLookup = false;
+bool fNameLookup = true;
```

Comment:
This appears to be a change of behaviour snuck into a refactoring PR.

Reply:
[It was always true](https://github.com/bitcoin/bitcoin/pull/6961#discussion_r44414684). Specifically, the global was initialized to false, but the default value was true. The upstream PR condensed this behaviour.

Reply:
LOL